pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/db/dao/FormActivityDaoTest.java

```diff
@@ -165,10 +165,11 @@ public void testInsertActivity_booleanQuestion() {
                 header,
                 footer,
                 rules,
+                true,
+                false,
                 yesOpt,
                 noOpt,
-                true,
-                false);
+                null); // Use the default render mode
```

Comment:
What is the default render mode?

Reply:
`RADIO_BUTTONS`. There will be a comment added to the `BooleanQuestionDef` constructor noting this behavior.